nabla-pendulum/deploy/07_configure_swap_pools.ts

```diff
@@ -0,0 +1,27 @@
+import { WasmDeployEnvironment } from "../../src/index";
+import { setInsuranceWithdrawalTimelock, setPoolCap, setSwapFees } from "../_lib";
+
+async function ConfigureSwapFees({ getNamedAccounts, deployments }: WasmDeployEnvironment) {
+  const { deployer } = await getNamedAccounts();
+
+  for (const swapPool of ["swap-usdc", "swap-eurc", "swap-brl"]) {
+    await setSwapFees(
+      deployments,
+      { from: deployer, log: true },
+      swapPool,
+      600, // 0.06% = 0.1% * 60%
+      400, // 0.04% = 0.1% * 40%
+      0 // 0%
+    );
+
+    await setInsuranceWithdrawalTimelock(deployments, { from: deployer, log: true }, swapPool, 7200);
+  }
+
+  await setPoolCap(deployments, { from: deployer, log: true }, "swap-usdc", 1000n * 10n ** 6n);
```

Comment:
Are the caps also intentionally different from what is defined [here](https://www.notion.so/satoshipay/24-05-16-Nabla-Pendulum-Deployment-c7cefd58f44c428a92ceaea30c603286?pvs=4#50914c2b39924ef3848d6580e868d795)?

Reply:
I have the same question. I checked the parameter definition for `maxTokens` and it look like this:
/// timelock in blocks
  maxTokens: bigint
  
I assume that the comment was left there by mistake and the parameter should indeed reflect an amount?

Reply:
Same as the comment above.